src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -343,3 +343,14 @@ def load_arguments(self, _):
         c.argument('intelligence_pack_name', options_list=['--name', '-n'])
         c.argument('workspace_name', options_list='--workspace-name')
     # endregion
+
+    # region monitor clone
+    with self.argument_context('monitor clone') as c:
+        c.argument('source_resource', help="Resource ID of the source resource.")
+        c.argument('target_resource', help="Resource ID of the target resource.")
+        c.argument('always_clone', action='store_true',
+                   help="If this argument is applied, "
+                        "all monitor settings would be cloned instead of expanding its scope.")
+        c.argument('monitor_types', options_list=['--types', '-t'], arg_type=get_enum_type(['metricsAlert']),
+                   nargs='+', help='List of types of monitor settings which would be cloned.', default=['metricsAlert'])
```

Comment:
No enum class?

Reply:
no. It's decided by me. we can define an enum class for it. But I think list of string is enough.

Reply:
what's the purpose of this value?

Reply:
currently, we just support clone the metric alerts. We may support clone more monitor settings in the future. This is used to control the kinds of the monitor settings which would be cloned by this command.

Reply:
Usually, the option enum value is capitalized. Is it better to follow the convention?

Reply:
Sure